### PR TITLE
[ALLUXIO-3265] Use static imports for standard test utilities for class SleepingTimerTest

### PR DESCRIPTION
**Pull request by ji-xu-guo**
[https://alluxio.atlassian.net/browse/ALLUXIO-3265](url)
change Mockito.verify(...) to verify(...) for all function of class alluxio.heartbeat.SleepingTimerTest and change import org.mockito.Matchers; to import static org.mockito.Matchers.verify;

Mockito.mock(...), Mockito.times(...) ,Mockito.any(...) and Mockito.anyString() are the same.

### CODE REVIEW

**Comment by apc999** [comment]
@ji-xu-guo, can you sign the SLA (link)

File: core/common/src/test/java/alluxio/heartbeat/SleepingTimerTest.java [hunk 1]
```diff
@@ -14,13 +14,21 @@
...
+import static org.mockito.Mockito.verify;
+
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.Mockito;
 import org.slf4j.Logger;
 
 import java.time.Duration;
 
+
```
Comment (apc999) [inline core/common/src/test/java/alluxio/heartbeat/SleepingTimerTest.java]: unnecessary empty lines. can you remove them?

File: core/common/src/test/java/alluxio/heartbeat/SleepingTimerTest.java [hunk 2]
```diff
@@ -14,13 +14,20 @@
...
+import static org.mockito.Mockito.verify;
+
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.Mockito;
 import org.slf4j.Logger;
-
 import java.time.Duration;
 
+
```
Comment (apc999) [inline core/common/src/test/java/alluxio/heartbeat/SleepingTimerTest.java]: can you remove line 29  and 30, rather than line 21?

**Comment by aaudiber** [comment]
@apc999 fyi merging this PR broke the build (fixed now). The build is quite fast and stable these days, so we should try to only merge PRs after the build passes

**Pull request merged**